akka-cluster-sharding/src/main/scala/akka/cluster/sharding/internal/EntityPassivationStrategy.scala

```diff
@@ -225,8 +277,9 @@ private[akka] final class LeastFrequentlyUsedEntityPassivationStrategy(
   }
 
   override def entityTouched(id: EntityId): PassivateEntities = {
+    val passivated = passivateExcessEntities(adjustment = +1)
     frequencyList.update(id)
-    passivateExcessEntities(skip = OptionVal.Some(id)) // make sure the newly created entity is still active
+    passivated
```

Comment:
My previous merge broke this, so now I push the correct merge (see https://github.com/akka/akka/pull/30951/commits/9de6039dc7ffeddf8dff36e81d8c888b63ce408d). 

However, the tests are failing for this. So not sure if that's what we need. I need to review the full PR to get a better understanding of the failure now.

Reply:
It's a different failure from earlier, and will be because the two approaches don't work together now. The adjustment assumes that this is always a new entity being created, but now that it's only using entity touched, it may be a new entity or an already active entity.

Reply:
Fixed, with comments, in de69dfc132ed8f093f523b8e3b000307bb377d82